vision/ui/main_window.py

```diff
@@ -0,0 +1,67 @@
+from tkinter import*
+ 
+# creating button for autonomous mapping
+window = Tk()
+window.title("welcome")
+window.geometry('300x300')
+window.configure(width =50,height = 50, bg='blue')
+ 
+lbl = Label(window, text = "Autonomous Mapping",  bg = 'red', fg ='white')
+lbl.grid(column=0, row=0)
+def clicked():
+    lbl.configure(text="Autonomous Mapping (Clicked)")
```

Comment:
I think it would be best to not change the label text when these are pressed. So for now you can replace all these with `pass` which just tells python to do nothing when it runs this method. Then, once the other code gets merged in we can call the appropriate methods.